Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.12-alpine as builder
+FROM golang:1.15-alpine as builder
```

Comment:
Any reason not to go to version 1.16 here?

Reply:
Nope, no reason 👍